src/taco-cli/cli/utils/cordovaHelper.ts

```diff
@@ -280,12 +281,19 @@ class CordovaHelper {
         });
     }
 
+    public static ensureCordovaVersionAcceptable(cliVersion: string): void {
+        if (semver.valid(cliVersion) && semver.lt(cliVersion, "5.4.0") && semver.gte(process.versions.node, "5.0.0")) {
```

Comment:
There is similar logic in taco-remote-lib/common/builder.ts. Wouldn't it be better to have a single source of truth for checking the known good range of Cordova versions?